Sources/SpeziValidation/ValidationState/ValidationContext.swift

```diff
@@ -17,7 +17,7 @@ import SwiftUI
 ///
 /// You can use this structure to retrieve the state of all ``ValidationEngine``s of a subview or manually
 /// initiate validation by calling ``validateSubviews(switchFocus:)``. E.g., when pressing on a submit button of a form.
-public struct ValidationContext {
+public struct ValidationContext: Sendable {
```

Comment:
I also looked into such a change as well when implementing #48.
However, I came to the conclusion that the `ValidationContext` is not `Sendable` (I don't have the necessary domain knowledge of SpeziAccount to make such a change). Now, with #50 being implemented by the SpeziViews god @Supereg himself (marking the required properties of `CapturedValidationState` as `nonisolated`) , this `Sendable` annotation should be fine from my perspective, enabling your clean preference closure implementation.

However, @Supereg should definitely make the final call here.